dev/breeze/setup.cfg

```
@@ -58,6 +58,7 @@ install_requires =
     pytest-xdist
     rich
     click_completion
+    requests
```

```suggestion
    requests
```

I think it can be removed :).

---

> @potiuk without "request" in the setup.cfg, we will have this error, I think it is necesary.

Indeed :). Merging